defaults.go

```diff
@@ -253,7 +253,18 @@ func IsMsg(buf []byte) error {
 
 // IsFqdn checks if a domain name is fully qualified.
 func IsFqdn(s string) bool {
-	return strings.HasSuffix(s, ".")
+	s2 := strings.TrimSuffix(s, ".")
```

Comment:
I still don't really like the changes. Having a new string and then comparing the entire thing again, while we're only interested in the last bit.

Reply:
`strings.TrimSuffix` only slices the string, it doesn’t create a new one. Do you have any alternative in mind?

Reply:
how is that not a new string? https://play.golang.org/p/d4-FIn187I6

Reply:
ok cool. Thanks for the explainer.

lgtm to me then